src/PressCenters.Common/TempDataConstants.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace PressCenters.Common
+{
+    public static class TempDataConstants
+    {
+        public const string Redirected = "Redirected";
```

Comment:
Declare this constant in `ContactsController`

Reply:
Thanks for the feedback!

Reply:
And last one: may be rename the name of the temp data to `"RedirectedFromContactForm"`

Reply:
Done. Sorry for wasting your time, but this was my first contribution to other project 😄.